config.txt

```diff
@@ -663,6 +663,27 @@
 
 # grab_mouse = 0
 
+# Set to 1 to use a save slot system (similar to emulator save states)
+# instead of allowing for freeform text entry. This value can be overridden
+# using a game.cnf file.
```

Comment:
I realize I approved this before, but once a game.cnf changes the config, the config is permanently modified for all subsequent games loaded. This means a game that sets this could effectively make the save slots for other games disappear or would globally disable save slots for other games if it wanted to. This behavior is pretty terrible but hasn't been fixed yet, so I don't think enabling these settings for game.cnf is a good idea for the time being.

Reply:
I just ran into this as well. I'll change the comments on config.txt and update configure.* accordingly in the meantime. It'll be easy to make these configurable per-game later on.

Reply:
Corrected in 8f9401a.